src/lib/src/header/navuser.component.css

```diff
@@ -3,9 +3,8 @@
  */
 .spin-holder {
   position: relative;
```

Comment:
fixes positioning of spinner https://github.com/PRX/styleguide.prx.org/pull/111/files/2215e8ec9bf0a3625aca8a8e5b3440b6427aeb94#r192900485